index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles/styles.css">
+    <title>Fabricio La Torre</title>
+</head>
+<body>
+    <header class="main-header">
+        <div class="container">
+            <h1 class="mh-logo">
+              <a href="#">La Torre</a>
+                
+            </h1>
+            <!--
+            <nav class="main-nav">
+               
+                <ul class="main-nav-list">
+                  
+                    <li>
+                        <a href="#">Portfolio</a>
+                    </li>
+                    <li>
+                        <a href="#">About</a>
+                    </li>
+                    <li>
+                        <a href="#">Contact</a>
+                    </li>
+                    <li>
+                        <a href="#"><img alt="maillogo" src="images/mail.svg"></a>
+                    </li>
+                </ul>
+            </nav>
+          -->
+              <h1 class="button">
+                  <a href="#"><img alt="hambutton" src="images/hambutton.svg"></a>
+              </h1>
+        </div>
+
+    </header>
+
+
+    <section class="body-container">
+   
+
+      <ul class="intro">
+        <li>
+          <h1 class="name">
+              Hey there! I'm Fabricio La Torre.
+          </h1>
+          <h3>
+              I'm a Web Developer 
+          </h3>
+          <p>
+              Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nobis doloremque ea sequi ullam dolor animi, cumque ab voluptate iusto sit.
+          </p>
+        </li>
+      
+        
+      </ul>
+      <ul class="socials">
+        <li><a href='https://github.com/FabricioLatl/'><img alt="github" src='images/github.svg' /></a></li>
+        <li><a href='https://www.linkedin.com/in/fabriciolatl/'><img alt="linkedin" src='images/linkedin.svg' /></a></li> 
+        <li><a href='https://twitter.com/Wafer_lt'><img alt="twitter" src='images/twitter.svg' /></a></li>
+      </ul>
```

Comment:
![all-icons](https://user-images.githubusercontent.com/93039359/177863836-f3e66ba1-3519-4ff8-9e85-accd0b4b6665.png)

[Optional]

- [ ] Please add all the icons as shown in the Figma template :+1: because your portfolio should contain all your social media links. :+1: by which recruiters will contact you :eyes: